policy/modules/system/systemd.te

```diff
@@ -1021,16 +1051,20 @@ files_search_runtime(systemd_logind_t)
 files_getattr_all_tmpfs_files(systemd_logind_t)
 files_rw_runtime_dirs(systemd_logind_t)
 
+fs_getattr_autofs(systemd_logind_t)
 fs_getattr_cgroup(systemd_logind_t)
 fs_getattr_tmpfs(systemd_logind_t)
 fs_getattr_tmpfs_dirs(systemd_logind_t)
+fs_getattr_xattr_fs(systemd_logind_t)
```

Comment:
These should probably be replaced with `fs_getattr_all_fs()` (except `fs_getattr_tmpfs_dirs()`). Other systemd domains have a similar pattern.